### ISSUE TITLE
aurutils terminal coloring with systemd 256

### ISSUE THREAD

**Issue by christian-heusel** [aurutils/aurutils#1161]
With systemd 256 (the upcoming systemd release) it includes possibilities to color terminals, i.e. on `systemd-nspawn` and `run0`. When nspawn is now used in a program this very much looks like a visual bug IMO:

![image](https://github.com/aurutils/aurutils/assets/26827864/084d99ed-1858-4a9e-aaf1-f8b0fc6af558)

I have therefore disabled the behaviour in devtools (https://gitlab.archlinux.org/archlinux/devtools/-/commit/0e2b16b0ac63e33e32a6cb889bc4b047e0d451e9), but since I couldn't get the above trivial fix to work with aurutils I thought I'd open this issue as a heads-up.

Feel free to close as wont-fix if you want to keep the default behaviour.

**Comment by Earnestly** [comment]
Seems like an nspawn issue; the bleeding background colour appears to come from pacman.

**Comment by christian-heusel** [comment]
@Earnestly I'm not sure I follow, what do you mean?

**Comment by AladW** [comment]
I'm confused by the commit message:
```
Disable this bevaviour for pkgctl, so it stays active for arch-nspawn (for now).
```
but `arch-nspawn`, `makechrootpkg`, and `mkarchroot` use `src/lib/common.sh`. In turn, `aur-chroot` uses those three tools for chroot functionality.

**Comment by christian-heusel** [comment]
I forgot that they include `common.sh` so its a side-effect but fine by me 😆 Disregard this issue then 👍🏻

**Comment by VorpalBlade** [comment]
Just noticed this. It looks like eye bleed unfortunately. Any way to get this fixed in aurutils?

**Comment by christian-heusel** [comment]
This is fixed in the devtools 1.2.1 release thats currently still in testing 😊
